spatialos-sdk/src/worker/core/parameters.rs

```diff
@@ -44,6 +59,46 @@ pub struct ConnectionParameters {
 }
 
 impl ConnectionParameters {
+    pub fn new(worker_type: &str) -> Self {
+        let mut params = ConnectionParameters::default();
+        params.worker_type = worker_type.to_owned();
+        params
+    }
+
+    pub fn with_protocol_logging(mut self, log_prefix: &str) -> Self {
+        self.enable_protocol_logging_at_startup = true;
+        self.protocol_logging.log_prefix = log_prefix.to_owned();
+        self
+    }
+
+    pub fn using_tcp(mut self, params: Option<TcpNetworkParameters>) -> Self {
```

Comment:
Not sure if I should split these into `using_tcp_default` and `using_tcp_with_params` instead of mandating an `Option` as an argument

Reply:
I think it should be just `using_tcp` with `TcpNetworkParameters::default()`. We should probably opt in to defaults.